### DISCUSSION TITLE
Rotate Selection

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [71/dance#263] [Q&A] [answered]
I just revisited the kakoune page and was intrigued by the swapping arguments example. I tried it in vscode but the actual rotation does not work with a-), is there maybe a different shortcut?

https://kakoune.org/why-kakoune/why-kakoune.html

**Comment by 71** [comment, marked as answer]
Looks like the key bindings for the "rotate" commands were reversed in Dance: `(` in Kakoune was `a-(` in Dance and `a-(` in Kakoune was `(` in Dance. Fixed in d7fc7e43ed3aecef49619943c153313185e60a9a. Thanks for the find!

**Reply by sschneider-ihre-pvs** [reply]
I don't seem to have any shortcut weather `a-)` nor `a-(`

**Reply by sschneider-ihre-pvs** [reply]
oh dang, alt+shift+9 of course. Anyway, that does not do anything

**Reply by sschneider-ihre-pvs** [reply]
It only swaps the selection and not the content.

**Reply by sschneider-ihre-pvs** [reply]
ah now I get it, nvm :D
